src/PdfReader/index.tsx

```diff
@@ -285,29 +338,59 @@ export default function usePdfReader(args: ReaderArguments): ReaderReturn {
   return {
     isLoading: false,
     content: (
-      <Flex
-        as="main"
-        zIndex="base"
-        flex="1 0 auto"
-        justifyContent="center"
-        alignItems="center"
-        tabIndex={-1}
-        id="iframe-wrapper"
-      >
-        <Document file={state.file} onLoadSuccess={onDocumentLoadSuccess}>
-          {isParsed && state.numPages && (
-            <>
-              {state.isScrolling &&
-                Array.from(new Array(state.numPages), (index) => (
-                  <Page key={`page_${index + 1}`} pageNumber={index + 1} />
-                ))}
-              {!state.isScrolling && (
-                <Page pageNumber={state.pageNumber} loading={<></>} />
-              )}
-            </>
-          )}
-        </Document>
-      </Flex>
+      <>
+        <Flex
+          as="main"
+          zIndex="base"
+          flex="1 0 auto"
+          justifyContent="center"
+          alignItems="center"
+          tabIndex={-1}
+          id={IFRAME_WRAPPER_ID}
+        >
+          <Document file={state.file} onLoadSuccess={onDocumentLoadSuccess}>
+            {isParsed && state.numPages && (
+              <>
+                {state.isScrolling &&
+                  Array.from(new Array(state.numPages), (_, index) => (
+                    <Page
+                      key={`page_${index + 1}`}
+                      width={width}
+                      pageNumber={index + 1}
+                    />
+                  ))}
+                {!state.isScrolling && (
+                  <Page
+                    pageNumber={state.pageNumber}
+                    width={width}
+                    loading={<></>}
+                  />
+                )}
+              </>
+            )}
+          </Document>
+        </Flex>
+
+        {/* TODO: Visual styling */}
+        {state.isScrolling && (
+          <Flex
+            as="footer"
+            position="sticky"
+            bottom={0}
+            left={0}
+            right={0}
+            height={`${HEADER_HEIGHT}px`}
+            zIndex="sticky"
+          >
+            <HStack ml="auto" spacing={1}>
+              <Button onClick={() => previousChapter()}>
+                Previous Chapter
+              </Button>
+              <Button onClick={() => nextChapter()}>Next Chapter</Button>
+            </HStack>
+          </Flex>
+        )}
```

Comment:
Oh interesting approach to add them here. These should be outside of this component though in the manager UI with the page buttons etc. You'd then add a `previousChapter` and `nextChapter` (or `previousResource` more accurately) api to the `navigator`. I think we should decide if we are going to have these buttons though first

Reply:
I thought because the chapter buttons would only appear in PDFs, they should go in a PDF specific place - so that epubs wouldn't be tempted to use them.

Reply:
We will need them for EPUBs as well if we choose not to show the forward/back button on scroll mode. EPUBs effectively work the same way as PDFs here - you still have to navigate to the next/prev resource using a button in scroll mode

Reply:
ah good point.  I will repurpose these buttons, that makes sense.